Fails to build on rustc 1.25.0-nightly (97520ccb1 2018-01-21)

```
error[E0050]: method `late_callback` has 7 parameters but the declaration in trait `rustc_driver::CompilerCalls::late_callback` has 8
  --> src/driver.rs:66:16
   |
66 |         ofile: &Option<PathBuf>,
   |                ^^^^^^^^^^^^^^^^ expected 8 parameters, found 7
   |
   = note: `late_callback` from trait: `fn(&mut Self, &rustc_trans_utils::trans_crate::TransCrate, &getopts::Matches, &rustc::session::Session, &rustc::middle::cstore::CrateStore, &rustc::session::config::Input, &std::option::Option<std::path::PathBuf>, &std::option::Option<std::path::PathBuf>) -> rustc_driver::Compilation`
```

--- a/src/driver.rs
+++ b/src/driver.rs
@@ -9,9 +9,11 @@ extern crate rustc;
 extern crate rustc_driver;
 extern crate rustc_errors;
 extern crate rustc_plugin;
+extern crate rustc_trans_utils;
 extern crate syntax;
 
 use rustc_driver::{driver, Compilation, CompilerCalls, RustcDefaultCalls};
+use rustc_trans_utils::trans_crate::TransCrate;
 use rustc::session::{config, Session};
 use rustc::session::config::{ErrorOutputType, Input};
 use std::path::PathBuf;
@@ -58,6 +60,7 @@ impl<'a> CompilerCalls<'a> for ClippyCompilerCalls {
     }
     fn late_callback(
         &mut self,
+        trans_crate: &TransCrate,
         matches: &getopts::Matches,
         sess: &Session,
         crate_stores: &rustc::middle::cstore::CrateStore,
@@ -66,7 +69,7 @@ impl<'a> CompilerCalls<'a> for ClippyCompilerCalls {
         ofile: &Option<PathBuf>,
     ) -> Compilation {
         self.default
-            .late_callback(matches, sess, crate_stores, input, odir, ofile)
+            .late_callback(trans_crate, matches, sess, crate_stores, input, odir, ofile)
     }
     fn build_controller(&mut self, sess: &Session, matches: &getopts::Matches) -> driver::CompileController<'a> {
         let mut control = self.default.build_controller(sess, matches);

